src/Skia/Avalonia.Skia/FormattedTextImpl.cs

```diff
@@ -590,7 +590,7 @@ private void Rebuild()
             else
             {
                 var lastLine = _skiaLines[_skiaLines.Count - 1];
-                _size = new Size(maxX, lastLine.Top + lastLine.Height + lastLineDescent);
```

Comment:
i think here we should use 
`
_size` = new Size(maxX, lastLine.Height);
`
as the width need to be the maximum width of the all lines, otherwise in case of wordwrap we can have measuring problems